Anlab.Mvc/Views/Lab/Confirmation.cshtml

```diff
@@ -6,7 +6,7 @@
 }
 <div class="col">
     @Html.Partial("_OrderDetails")
-    <form asp-controller="Lab" asp-action="Confirmation" asp-route-id="@Model.Order.Id" method="post">
+    <form id="form" asp-controller="Lab" asp-action="Confirmation" asp-route-id="@Model.Order.Id" method="post">
```

Comment:
@jSylvestre this switch method is ok but it's also easy in the future if you just make 2 forms, then no javascript needed

Reply:
@srkirkland There is form data there. If there were 2 forms would that data be included in both?

Reply:
@jSylvestre yea

Reply:
@jSylvestre maybe not the best option for this specific situation